Library/Homebrew/utils/shared_audits.rb

```diff
@@ -70,28 +70,20 @@ def github_release(user, repo, tag, formula: nil, cask: nil)
   def gitlab_repo_data(user, repo)
     @gitlab_repo_data ||= {}
     @gitlab_repo_data["#{user}/#{repo}"] ||= begin
-      out, _, status= curl_output("--request", "GET", "https://gitlab.com/api/v4/projects/#{user}%2F#{repo}")
-      return unless status.success?
-
-      JSON.parse(out)
+      out, _, status = curl_output("https://gitlab.com/api/v4/projects/#{user}%2F#{repo}")
+      JSON.parse(out) if status.success?
```

Comment:
This will now cache `nil`, while the previous version didn't since it returned when `curl` failed.

Reply:
There's no change in behavior as an unassigned instance variable `@variable` is still nil. (I tested this in `brew irb`)